code/game/objects/items/devices/PDA/cart.dm

```diff
@@ -88,7 +88,7 @@
 /obj/item/cartridge/detective
 	name = "\improper D.E.T.E.C.T. cartridge"
 	icon_state = "cart-s"
-	access = CART_SECURITY | CART_MEDICAL | CART_MANIFEST
+	access = CART_SECURITY | CART_MEDICAL | CART_MANIFEST | CART_REAGENT_SCANNER
```

Comment:
Why?

Reply:
This is redundant because the forensic scanner already shows reagent contents.

Reply:
Yea, exactly. Figured, why not? Makes det a little bit less reliant on his scanner.